app/views/catalog/_show_actions_box_default.html.erb

```diff
@@ -15,14 +15,33 @@
     <% end %>
   </div>
 
-  <div class='mr-auto'>
-    <%= render partial: 'catalog/bookmark_control', locals: { document: document } %>
-  </div>
+  <div class='mr-auto al-show-actions-box-options'>
+    <% if requestable %>
+      <div class='mr-auto al-show-actions-box-request'>
+          <%= render partial: 'arclight/requests/google_form', locals: { document: document } %>
+      </div>
+    <% end %>
 
-  <% if requestable %>
-    <div class='mr-auto al-action-box-request'>
-        <%= render partial: 'arclight/requests/google_form', locals: { document: document } %>
+    <div class='mr-auto al-show-actions-box-bookmarks'>
+      <%= render partial: 'catalog/bookmark_control', locals: { document: document } %>
     </div>
-  <% end %>
+  </div>
+
+    <% if document.downloads.files.present? %>
```

Comment:
Thanks for taking care of this James.  One thing that I think I may not have been clear on, is that I think the content that is here in this partial to render the download links can actually be the content of the `document_downloads` partial instead.  Then in this partial, we can simply call `render document.downloads`.  The split button stuff that is in that partial currently isn't necessary w/ the new designs so that can be dropped and that partial can just contain the link rendering markup below.

Reply:
I'm extremely sorry for that error, this has been adjusted.